docs/en/status-communities/transfer-your-community-s-ownership.md

```diff
@@ -8,9 +8,36 @@ hide:
 
 # Transfer your community's ownership
 
- :octicons-tools-24: In progress
+:octicons-beaker-24: Beta
 
-!!! note ""
-     We're working on this content.
+You can transfer your Status Community ownership if you can't manage it and want to pass on the responsibility to someone else. To transfer your community's ownership, you need a [backup of the community's private key][back-up-your-community-s-private-key].
```

Comment:
I'd change the first sentence to something like 'If you can't manage your Status Community anymore, you can transfer the ownership to someone else'. Otherwise it can be a bit unclear what 'it' refers to in 'can't manage it'.